### PR TITLE
Add ap_data type to Phase 2 GMT output for GT interface

### PR DESCRIPTION
**Pull request by zhenbinwu**
#### PR description:

1. Added the ap_data type in the Phase2 GMT output dataformat, as requested by the GT group.
2. Update the GMT Z0/D0 LSB

### CODE REVIEW

**Comment by aloeliger** [comment]
Corresponds to https://github.com/cms-l1t-offline/cmssw/pull/1079

**Comment by aloeliger** [comment]
please test

**Comment by aloeliger** [comment]
please test

**Comment by aloeliger** [comment]
+l1

**Comment by rappoccio** [comment]
+1

File: DataFormats/L1TMuonPhase2/src/classes_def.xml [hunk 1]
```diff
@@ -8,8 +8,8 @@
  <class name="edm::Wrapper<std::vector<l1t::MuonStub > >"/>
  <class name="edm::Ref<l1t::MuonStubCollection>" splitLevel="0"/>
 
- <class name="l1t::TrackerMuon" ClassVersion="3">
-   <version ClassVersion="3" checksum="2397458791"/>
```
Comment (perrotta) [inline DataFormats/L1TMuonPhase2/src/classes_def.xml:12]: Do not remove previous versions!
```
   <version ClassVersion="3" checksum="2397458791"/>
```
must be left below `ClassVersion="4"` here below.

A fix is in order as soon as next 2300 IB will become available

Comment (aloeliger) [reply DataFormats/L1TMuonPhase2/src/classes_def.xml:12]: My apologies, I should have caught this.

Comment (perrotta) [reply DataFormats/L1TMuonPhase2/src/classes_def.xml:12]: Fix prepared in #41293: thank you @aloeliger !

**Pull request merged**